dace/libraries/blas/utility/memory_operations.py

```diff
@@ -0,0 +1,117 @@
+"""
+Helper functions for memory movements
+"""
+import dace
+from dace.memlet import Memlet
+import math
+from dace.libraries.blas.utility.initialization import *
+from dace import dtypes
+import numpy as np
+
+
+
+# ---------- ----------
+# NUMPY
+# ---------- ----------
+def aligned_ndarray(arr, alignment=64):
```

Comment:
There are two copies of this function now. Consider importing it from another place and using the new function in `tiling_vectorization_test.py` too

Reply:
Honestly, to me it seems very weird to import a function from a test into a file, which is part of the tool itself. Where else is this function? And if its nowhere else, why not import it in the test from here or put it into some other sensible place within dace itself and not the tests?